ansys/dpf/core/meshed_region.py

```diff
@@ -285,6 +285,10 @@ def set_coordinates_field(self, coordinates_field):
         coordinates_field : PropertyField or Field
         """
         self._api.meshed_region_set_coordinates_field(self, coordinates_field)
+        if self._full_grid is not None:
```

Comment:
@PProfizi In my opinion, we need to think larger. Here the "real" issue is not "when set coordinates is called the grid needs to be updated" but "when the mesh is updated (equivalent to setters are called) the grid is out dated".
With your we will have the same bug if set connectivity, or set element types is called